### PR TITLE
Community - Add support for timestamped embeds

### PR DESCRIPTION
**Pull request by netuoso**
Continuation of https://github.com/steemit/condenser/pull/3089 with twitch support merged in and tested.

### CODE REVIEW

**Comment by netuoso** [comment]
Ok for reducing complexity I am pushing a commit to only support `t=seconds` format. Although youtube itself does parse hours/minutes/seconds. I am not sure how common they actually are.

**Comment by netuoso** [comment]
@relativityboy ok reduced support to seconds only for each.

eg youtube: https://www.youtube.com/watch?v=QASbw8_0meM&t=5h
eg vimeo: https://vimeo.com/1084537#t=60s

also added more tests for variations of the youtube link and for the twitch embeds as well.

**Comment by netuoso** [comment]
So with the team slimming down, is there any point for continuing with community PR such as this one? I believe it was fairly well peer-reviewed already and tests were added. It is passing CI and works well on mspsteem.com

Should I close the PR/issue or just let it get orphaned in the backlog?

Just trying to decipher the state of things and determine if this is just noise for you guys.

File: src/shared/HtmlReady.test.js [hunk 1]
```diff
@@ -205,7 +205,52 @@ describe('htmlready', () => {
         const testString =
             '<html><p>before text https://vimeo.com/193628816/ after text</p></html>';
         const htmlified =
-            '<html xmlns="http://www.w3.org/1999/xhtml"><p>before text ~~~ embed:193628816 vimeo ~~~ after text</p></html>';
+            '<html xmlns="http://www.w3.org/1999/xhtml"><p>before text ~~~ embed:193628816 vimeo 0 ~~~ after text</p></html>';
```
Comment (roadscape) [inline src/shared/HtmlReady.test.js]: Start must to be optional, it's important the old tests continue to work as-is

**Comment by roadscape** [comment]
Hi @netuoso -- this is looking solid, main thing that I noticed was that old tests were modified to include start time and that is concerning unless we can be sure there are no side-effects.

**Comment by netuoso** [comment]
@roadscape please recheck

**Comment by roadscape** [comment]
Excellent, thanks! Moved to local PR #3288

**Pull request closed without merging**